support-frontend/assets/components/checkoutForm/checkoutForm.jsx

```diff
@@ -35,6 +35,35 @@ FormSection.defaultProps = {
   noBorder: false,
 };
 
+// Hidden version of form section
```

Comment:
is "hidden" really the defining feature of this? if it really is hidden, then there must be some other reason for it to exist